proto/frequenz/api/common/metrics/electrical.proto

```diff
@@ -8,7 +8,7 @@
 
 syntax = "proto3";
 
-package frequenz.api.common.electrical;
+package frequenz.api.common.metrics.electrical;
```

Comment:
For this packaging, the `electrical.proto` file needs to be in a metrics directory.

I'd suggest
1. renaming [`metrics.proto`](https://github.com/frequenz-floss/frequenz-api-common/blob/v0.x.x/proto/frequenz/api/common/metrics.proto) to `metric.proto`, and changing the package name there to `frequenz.api.common.metric`.
2. putting `electrical.proto` in a directory here named `metrics`.

Reply:
I see. This is what I was referring to in point 2.

Reply:
Actually, point 1 would mean that we use `metric::Metric` and `metrics::electrical::Dc/Ac`.
`metrics::Metric` would make be more consistent here.

There is nothing to be done for this comment.

I can approve once the CI passes.

Reply:
The CI doesn't pass for unrelated reasons (the main branch is also failing). I will create a separate PR with the fixes for the CI.

Reply:
I also enabled branch protection to require test to pass.